delta-lake/src/integration/java/org/apache/iceberg/delta/TestSnapshotDeltaLakeTable.java

```diff
@@ -454,7 +472,7 @@ private Dataset<Row> typeTestDataFrame() {
         .withColumn("structCol", expr("STRUCT(mapCol, arrayCol)"));
   }
 
-  private Dataset<Row> nestedDataFrame() {
+  private static Dataset<Row> nestedDataFrame() {
```

Comment:
this can be a static variable instead of a static method

Reply:
Thank you for your suggestion. I moved the dataframe initialization code blocks to `BeforeClass`.